cmake/KokkosConfigCommon.cmake.in

```diff
@@ -312,3 +312,18 @@ function(kokkos_compilation)
     endforeach()
   endif()
 endfunction()
+
+function(kokkos_set_launch_compiler)
+  cmake_parse_arguments(INP "" "Kokkos_DIR" "" ${ARGN})
+
+  if(Kokkos_USE_LAUNCH_COMPILER)
+    message(
+      STATUS
+        "kokkos_launch_compiler is enabled globally. C++ compiler commands with -DKOKKOS_DEPENDENCE will be redirected to the appropriate compiler for Kokkos unless they are listed in Kokkos_COMPILE_LAUNCHER_EXCLUDE_DIR CACHE variable."
```

Comment:
`Kokkos_COMPILE_LAUNCHER_EXCLUDE_DIR` is inconsistent.

Reply:
yeah, I am not really happy with the name, it is more of a placeholder ... any ideas?